src/GuestPages/Invite.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Container, Button } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const Invite = () => (
+  <>
+    <Container>Invite Page!</Container>
+    <Container>Main Image</Container>
+    <Container>Couple1 Name</Container>
+    <Container>&</Container>
+    <Container>Couple2 Name</Container>
+    <Container>Date</Container>
+    <Container>Time</Container>
+    <Container>Location</Container>
```

Comment:
Something to consider, but is not a required change before merging: Could Date, Time, and Location all be within one Container element and styled individually as needed from there? Feel free to disregard this comment if this logic would not work/apply.